db/deploy/util_functions/upsert_archive_trigger.sql

```diff
@@ -0,0 +1,53 @@
+-- Deploy ccbc:util_functions/upsert_archive_trigger to pg
+-- requires: trigger_function/set_archived
+
+begin;
+
+create or replace function ccbc_private.upsert_archive_trigger(
+  table_schema_name text,
+  table_name text,
+  user_table_schema_name text default 'ccbc_public',
+  user_table_name text default 'ccbc_user'
+)
+returns void as $$
+
+declare
+  column_string text;
+  index_string text;
+  comment_string text;
```

Comment:
Looks like some of these variables are also unused

Reply:
Removed